src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -647,6 +672,60 @@ emit_xconst_v128 (MonoCompile *cfg, MonoClass *klass, guint8 value[16])
 	return ins;
 }
 
+static guint64
+get_xconst_int_elem (MonoCompile *cfg, MonoInst *ins, MonoTypeEnum etype, int index)
+{
+	guint8 cns_vec[16];
+	if (ins->opcode == OP_XZERO) {
+		memset (cns_vec, 0x00, 16);
+	} else if (ins->opcode == OP_XONES) {
+		memset (cns_vec, 0xFF, 16);
+	} else {
+		g_assert (ins->opcode == OP_XCONST);
+		memcpy (cns_vec, ins->inst_p0, 16);
+	}
+	g_assert (index >= 0);
+	switch (etype) {
+		case MONO_TYPE_I1: {
```

Comment:
Nit: format issue: `case` should have the same indentation as `switch`.

Reply:
Fixed.